assets/wix/files.wxs

```diff
@@ -3153,6 +3147,57 @@
       <Component Id="cmp42A739F2AAC24D729DDBCAA096B8C55A">
         <File Id="fil48314E2AC49B4E208C5667CEC50CA78C" KeyPath="yes" Source="$(var.ProductSourcePath)\RegisterMicrosoftUpdate.ps1" />
       </Component>
+      <Component Id="cmp8AAE18271AD64A009017665757ED915B">
+        <File Id="filBD4DDF972FCA4F898D0ACABC42936DE6" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.SDK.pdb" />
+      </Component>
+      <Component Id="cmpB5AD8BDC2504469FADEFE3E6AA80BC27">
+        <File Id="fil6F9E70AACDD04A6587D0D7FB4C8D841E" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.Commands.Management.pdb" />
+      </Component>
+      <Component Id="cmpCCC817E4F2A04A73B4101A69417423A9">
+        <File Id="fil160973564B3C42FFA3C4AC9AF90F25CE" KeyPath="yes" Source="$(var.ProductSourcePath)\System.Management.Automation.pdb" />
+      </Component>
+      <Component Id="cmp8FDBF26F5E6F4BEB86E0FE5ED91CA638">
+        <File Id="fil806014EEC0C34C7F91B99267819124B8" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.WSMan.Runtime.pdb" />
+      </Component>
+      <Component Id="cmp074B6F8EA20548DCA2FA446D9E27D3FE">
+        <File Id="fil98B28DD6CBD34D758381CE290DB71B18" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.Security.pdb" />
+      </Component>
+      <Component Id="cmp41A0A813CB2C476389F59D10C2E7D5B5">
+        <File Id="fil46C395C891C74DFDB5014DB89142FFC9" KeyPath="yes" Source="$(var.ProductSourcePath)\pwsh.pdb" />
+      </Component>
+      <Component Id="cmpE3D0DEFE9F454D7DA5296449DD21D484">
+        <File Id="filBA7D37A15BD847B49C638EA974AA78AF" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.ConsoleHost.pdb" />
+      </Component>
+      <Component Id="cmpB7F854623ED34DB5B1B2C38D443BE731">
+        <File Id="filCEB5776695264EFB8EA506FB8135BD8F" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.Management.Infrastructure.CimCmdlets.pdb" />
+      </Component>
+      <Component Id="cmp5B1AAAB6B46D45EA8A430A766325B0D5">
+        <File Id="filF4983633E7914A1AAD5D02BFDFA7DD63" KeyPath="yes" Source="$(var.ProductSourcePath)\JsonSchema.Net.dll" />
+      </Component>
+      <Component Id="cmp61D33FB49B2E4C7D8E9C5EFC3F3A04B2">
+        <File Id="fil78E3FEFE7B0E4FE1993AC9C117B582C5" KeyPath="yes" Source="$(var.ProductSourcePath)\JetBrains.Annotations.dll" />
```

Comment:
I wonder we have to include the dependency.
I think that's the problem. I don't think that's what we want to distribute.

And what is official license for the file?

Reply:
https://www.nuget.org/packages/JetBrains.Annotations/

License is MIT.

I've had requests to remove this before.  It's actually a dependency.  I use `[RegexParam]` in a few of my methods where a Regex string is expected.  It provides Regex syntax highlighting in the string literal.  If there's a way to get this without the runtime dependency, I'm unaware of it.

Reply:
> It provides Regex syntax highlighting in the string literal.

You could use .Net attribute https://source.dot.net/#System.Text.RegularExpressions/System/Text/RegularExpressions/Regex.cs,59

Reply:
This will require an update to the library, so I'll have to update the version ref in the _.csproj_ file after I publish.  Shouldn't take long.

Reply:
That's only available in .Net 7 preview: https://docs.microsoft.com/en-us/dotNet/API/system.diagnostics.codeanalysis.stringsyntaxattribute?view=net-7.0&viewFallbackFrom=netframework-4.8)  My library is .Net Standard 2.0.

Reply:
From [docs](https://www.jetbrains.com/help/resharper/Code_Analysis__Annotations_in_Source_Code.html):
>By default, all annotation attribute classes are marked with [Conditional("JETBRAINS_ANNOTATIONS")] attribute so the compiler will ignore the attribute usages in your code, which means that no binary reference to the 'JetBrains.Annotations.dll' assembly will be produced. However, you can define 'JETBRAINS_ANNOTATIONS' conditional compilation symbol in your projects to preserve the attributes in the metadata.

So you can remove the dependency from binary package.

Reply:
That removes the attribute from the compilation. That doesn't help my users, which is the entire point of including the attribute.  I don't need it for my own development; it's so that the people who use the library get the benefit.

Reply:
You can use [embed attribute declarations](https://www.jetbrains.com/help/resharper/Code_Analysis__Annotations_in_Source_Code.html#embedding-declarations-of-code-annotations-in-your-source-code)